site/schemas/create-consequence-network.schema.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+import { Severity } from "../constants";
+import { setZodDefaultError, zodTimeInMinutes } from "../utils";
+
+export const createConsequenceNetworkSchema = z.object({
+    description: z.string(setZodDefaultError("Enter a consequence description")).min(1).max(500, {
+        message: "Description must not exceed 500 characters",
+    }),
+    removeFromJourneyPlanners: z.union(
+        [z.literal("yes"), z.literal("no")],
+        setZodDefaultError("Select planned or unplanned"),
+    ),
+    disruptionDelay: zodTimeInMinutes("Enter a valid time for disruption delay").optional().or(z.literal("")),
```

Comment:
Do we have some hint text or similar to help advise the user? If we are validating that the user has to enter between 0 and 999 minutes, this error message doesnt help them understand that.

Reply:
done